hooks/useEasyWrite.ts

```diff
@@ -20,9 +21,12 @@ export const useEasyWrite = (
   const error = prepareError || writeError;
   const isLoading = prepareIsLoading || writeIsLoading;
 
+  useEffect(() => {
+    if (writeData?.hash) notify({ hash: writeData?.hash, status });
+  }, [notify, writeData?.hash, status]);
```

Comment:
Does this need to be in a `useEffect`? Could it be an if statement outside of a `useEffect`?